src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -177,15 +185,38 @@ private CloudBlockBlob acquireLease(CloudBlockBlob cloudBlockBlob, String contai
             }
             cloudBlockBlob.acquireLease(blobLeaseTimeout, null);
             return cloudBlockBlob;
-        });
+        } catch (StorageException storageException) {
+            if (storageException.getHttpStatusCode() == HttpStatus.CONFLICT.value()) {
+                log.error(
+                    "Lease already acquired for container {} and zip file {}",
```

Comment:
keeping the ordering of the parameters in the logging the same could help read the logs.
Ever thought of putting this into a json object for logging ?
I guess that depends on if we catch our logs in json because then it becomes very easy to search and manipulate.